### PR TITLE
HPCC-26476 FTSlave cleanup

### PR DESCRIPTION
**Pull request by richardkchapman**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [x] Scalability
  - [x] Performance
  - [x] Security
  - [x] Thread-safety
  - [x] Cloud-compatibility
  - [x] Premature optimization
  - [x] Existing deployed queries will not be broken
  - [x] This change fixes the problem, not just the symptom
  - [x] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:
Compiled.

### CODE REVIEW

**Comment by richardkchapman** [comment]
@jakesmith @ghalliday 

Main concern would be if the removed functionality (a) might be needed in the near future or (b) is potentially required by clients from old systems connecting to new ones. I don't THINK that either are likely to be the case.

**Comment by jakesmith** [comment]
>Main concern would be if the removed functionality (a) might be needed in the near future or (b) is potentially required by clients from old systems connecting to new ones. I don't THINK that either are likely to be the case.

For (a) , I'd say even if it is, which I doubt, given I think this code has not been used in a long time, we can re-introduce what we need, and in the meantime the code is more maintainable without the noise of unused code.

For (b) I suspect that can't be true for any modern system, if it's in use, it's probably only in v. old versions.

The code changes/removal looks good to me.

**Review by jakesmith — APPROVED** [review]
Looks good to me

File: esp/services/ws_dfu/ws_dfuService.cpp [hunk 1]
```diff
@@ -2230,7 +2230,7 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor *udesc, co
         df->setRestrictedAccess(changeRestriction==CDFUChangeRestriction_Restrict);
     }
 
-    offset_t size=queryDistributedFileSystem().getSize(df), recordSize=df->queryAttributes().getPropInt64("@recordSize",0);
+    offset_t size=df->getFileSize(true, false), recordSize=df->queryAttributes().getPropInt64("@recordSize",0);
```
Comment (ghalliday) [inline esp/services/ws_dfu/ws_dfuService.cpp:2233]: @richardkchapman this isn't directly connected to removing the old code.  Although the code for the two calls looks identical, do you want to keep this change?

**Review by ghalliday — COMMENTED** [review]
@richardkchapman looks good - one question.

File: esp/services/ws_dfu/ws_dfuService.cpp (hunk 1 above)
Comment (richardkchapman) [reply esp/services/ws_dfu/ws_dfuService.cpp:2233]: It is required to remove the link to the dalift dll from esp services.

**Review by ghalliday — APPROVED** [review]

**Pull request merged**